viabel/optimization.py

```diff
@@ -142,6 +144,19 @@ def descent_direction(self, grad):
 
 class RMSProp(StochasticGradientOptimizer):
     """RMSProp optimization method
+    
+    Parameters
+    -----------
+    beta : `float` optional
+        hyper-parameter. The default is 0.9
```

Comment:
It would be good to describe the algorithm in the comment so the beta hyperparameter makes sense. Same with the other optimization classes